web/BlazorBuddies.Web/BlazorBuddies.Web.csproj

```diff
@@ -25,4 +25,8 @@
     <Watch Include="**\*.css" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Needed with line 24 above?

Reply:
Nope. Silly Visual Studio making changes without ensuring there wasn't another ItemGroup that handled the file already. I've removed it.